config/kubernetes/production/ingress.yml

```diff
@@ -134,6 +134,10 @@ metadata:
       SecRuleEngine On
       SecRequestBodyLimit 15728640
       SecRequestBodyNoFilesLimit 1048576
+      SecRule TIME_WDAY "!@pm 1 2 3 4 5" \
+        "id:997,phase:1,deny,status:503,tag:github_team=laa-crime-apply"
+      SecRule TIME_HOUR "!@pm 06 07 08 09 10 11 12 13 14 15 16 17" \
```

Comment:
Unless I'm misunderstanding how the rule works, shouldn't we be excluding "06" and including "18" if the hours are 7:00-19:00?

Reply:
I'm pretty sure server time is UTC

Reply:
Ah, of course.